providers/edge3/src/airflow/providers/edge3/version_compat.py

```diff
@@ -33,3 +33,15 @@ def get_base_airflow_version_tuple() -> tuple[int, int, int]:
 
 
 AIRFLOW_V_3_0_PLUS = get_base_airflow_version_tuple() >= (3, 0, 0)
+AIRFLOW_V_3_1_PLUS: bool = get_base_airflow_version_tuple() >= (3, 1, 0)
+
+if AIRFLOW_V_3_0_PLUS:
+    from airflow.sdk import BaseOperator
+else:
+    from airflow.models import BaseOperator
+
+__all__ = [
+    "AIRFLOW_V_3_0_PLUS",
+    "AIRFLOW_V_3_1_PLUS",
+    "BaseOperator",
+]
```

Comment:
Ditto here
```suggestion

try:
    from airflow.sdk import BaseOperator
except ImportError:
    from airflow.models import BaseOperator

__all__ = [
    "AIRFLOW_V_3_0_PLUS",
    "BaseOperator",
]
```

Reply:
A few more places have this pattern that I haven't commented on

Reply:
I get the idea but we still need 

```
AIRFLOW_V_3_1_PLUS: bool = get_base_airflow_version_tuple() >= (3, 1, 0)
```

in order to check whether to use the `DualStatsManager` or the original code. Otherwise there are compatibility issues because the `DualStatsManager` doesn't exist in older providers.